src/main/java/com/hivemq/swarm/extension/sdk/metrics/MetricsConsumerInput.java

```diff
@@ -0,0 +1,7 @@
+package com.hivemq.swarm.extension.sdk.metrics;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface MetricsConsumerInput {
+    void createMaps();
```

Comment:
Here you want the methods to receive the actual values. 

Like: `getTimerMap()` etc.